src/Core/Repositories/SqlServer/CipherRepository.cs

```diff
@@ -629,7 +629,7 @@ private DataTable BuildCiphersTable(SqlBulkCopy bulkCopy, IEnumerable<Cipher> ci
             ciphersTable.Columns.Add(creationDateColumn);
             var revisionDateColumn = new DataColumn(nameof(c.RevisionDate), c.RevisionDate.GetType());
             ciphersTable.Columns.Add(revisionDateColumn);
-            var deletedDateColumn = new DataColumn(nameof(c.DeletedDate), c.DeletedDate.GetType());
+            var deletedDateColumn = new DataColumn(nameof(c.DeletedDate), typeof(DateTime));
```

Comment:
Is this an issue because it is nullable (vs .creation and revision dates)?

Reply:
Correct.